packages/web/src/components/coinflow-onramp-modal/CoinflowOnrampModal.tsx

```diff
@@ -74,6 +74,7 @@ export const CoinflowOnrampModal = () => {
     >
       {showContent ? (
         <CoinflowPurchase
+          email={guestEmail}
```

Comment:
is this supposed to be the account email if you're not in guest mode?
or is this only used for guest checkout?

Reply:
if not in guest mode, this will be empty so it'll prompt the user for the email in the coinflow modal. if you've already purchased before, coinflow will remember your email and won't prompt you again for it. good callout! i tested this a couple different ways logged in, new account, with/without.